web/themes/custom/cloudy/src/components/message/message.scss

```diff
@@ -2,42 +2,28 @@
   &--#{$class},
   &-#{$class} {
     background-color: $color1;
-    border-color: $color2 !important; // BAD, but only way to override Bootstrap
     color: $color2;
 
-    a {
+    &__heading {
       color: $color2;
     }
   }
 }
 
 .message {
   display: block;
-  margin-bottom: 2px;
   padding: $cloudy-space-4;
-  color: $cloudy-color-neutral-0;
+  color: $cloudy-color-neutral-900;
+  border-radius: $cloudy-space-0;
+  border: none !important; // BAD, but only way to override Bootstrap
```

Comment:
Do you need the !important if you aren't using a Boostrap class anymore?

Reply:
Nope. It's gone :)